makefiles/devel.mk

```diff
@@ -285,7 +288,7 @@ hpp-statistics.configure.dep: hpp-util.install hpp-statistics.checkout
 hpp-core.configure.dep: example-robot-data.install hpp-constraints.install \
 	hpp-statistics.install hpp-core.checkout
 hpp-constraints.configure.dep: hpp-pinocchio.install hpp-statistics.install \
-	hpp-constraints.checkout
+	hpp-environments.install hpp-constraints.checkout
```

Comment:
Why do you need hpp-environments here ?

Reply:
Because a unit test needs Romeo description. I guess the unit test could be made independent of this robot but this is another topic.

Reply:
Yes, I guess that is the explanation here.
When I added dependencies, it was because packages asked for them. I did not checked in depth why they asked for those dependencies.